app/controllers/case_management/clients_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
       # Don't create additional clients if we already have one
       return redirect_to case_management_client_path(id: intake.client_id) if intake.client_id.present?
 
-      client = Client.create_from_intake(intake)
+      client = Client.create!
```

Comment:
could also do Client.create!(intake: intake) which will associate the intake object to the client on creation, and remove line 22.

Reply:
Also, if we are leaving vita_partner_id on client, we def should create the client with the vita_partner_id of the intake!

```Client.create!(intake: intake, vita_partner_id: intake.vita_partner_id)```

Reply:
Thanks! we did exactly as you suggested, good catch